gitpod-io/website
# Update content-security-policy to support Dedicated (*.gitpod.cloud)

## Description


Without this change, Content Security Policy prevents you from using the Simple Browser extension to preview website changes.

## Related Issue(s)



Fixes # n/a

## How to test



Open a Gitpod Workspace from the content of this PR, from ports for port 3000, click Open Preview. Instead of seeing a "sad mac" icon, you should see the preview for Gitpod.

## Release Notes



```release-note
NONE
```
